pkg/statsreceiver/metadata.go

```diff
@@ -0,0 +1,43 @@
+package statsreceiver
+
+import (
+	"github.com/snowplow/snowbridge/pkg/models"
+	"net/http"
+)
+
+type MetadataReporterConfig struct {
+}
+
+type metadataReporter struct {
+	client *http.Client
+}
+
+func NewMetadataReporter(config MetadataReporterConfig) *metadataReporter {
+	return &metadataReporter{}
+}
+
+func (s *metadataReporter) Send(b *models.ObserverBuffer) {
+	metadata := b.ErrorsMetadata
+	doSomeMagic(metadata.Invalid)   // it's a list now, so aggregate/group with counts here (or do it early when appending write results in a buffer)
+	doSomeMagic(metadata.Retryable) // same here
+
+	/* TODO how should event we send here look like?
```

Comment:
@jbeemster I'd love to take a read from you on this, as you understand the end to end system well. Any thoughts on how to structure data coming through the metadata system?

Reply:
https://www.notion.so/keep-in-the-snow/Snowbridge-metadata-metrics-reporting-21b07af295a28010af25dff43b628093

^^ Schema proposal